helm/hpcc/templates/_helpers.tpl

```diff
@@ -138,17 +150,39 @@ Add ConfigMap volume for a component
 {{- end -}}
 
 {{/*
-Add data volume mount
-If any storage planes are defined that name pvcs they will be mounted
+Returns a non empty string if any labels in the list includeLabels is in the plane.labels
+Note: the list includeLabels may contain an empty string (""), which matches planes that do not have a label
+Pass in plane and includeLabels
+Return: If there is any matching labels, there will be a non-empty string returned.  If there is no matching labels,
+        an empty string will be returned.
 */}}
-{{- define "hpcc.addDataVolumeMount" -}}
+{{- define "hpcc.doesStorageLabelsMatch" -}}
+{{- $plane := .plane -}}
+  {{- range $label := .includeLabels -}}
+    {{- if and (eq $label "") (not $plane.labels) -}}
+      {{- print "true" -}}
```

Comment:
minor: might be slightly clearer that non-null means a match if it returned "T" instead of true.